src/Vlingo.Actors/Vlingo.Actors.csproj

```diff
@@ -6,7 +6,7 @@
 
     <!-- NuGet Metadata -->
     <IsPackable>true</IsPackable>
-    <PackageVersion>0.4.10</PackageVersion>
```

Comment:
Don't bump the version. we will release this next time.

Reply:
Got it, no more confirmed bumps